README.md

```diff
@@ -14,6 +14,22 @@ npm run build
 
 ## Deployment
 
+### Automated Deployment
+
+We now have a series of scripts that will help with automating the release process. To use these scripts, do the following.
+
+1. Run `npm install` in the project root to make sure all dependencies are installed at the project level and in the `./scripts` folder.
+1. Run `npm run bump-version` in the project root. This will increment the project version and create a git commit.
+1. Run `node scripts/commands/readme.js` in the project root. This will prompt you to select the commits that are part of this release. The commit log messages will be automatically added to the [changelog](readme.txt) and committed.
+1. Manually create a new PR with the updated version and change log.
+1. Once that PR is approved and merged, run `npm run create-release`. Follow the prompts to do a build, create a new release tag, and deploy the release tag.
```

Comment:
Can we use human-friendly numbering so we aren't dependent on GitHub's rendering?
```suggestion
2. Run `npm run bump-version` in the project root. This will increment the project version and create a git commit.
3. Run `node scripts/commands/readme.js` in the project root. This will prompt you to select the commits that are part of this release. The commit log messages will be automatically added to the [changelog](readme.txt) and committed.
4. Manually create a new PR with the updated version and change log.
5. Once that PR is approved and merged, run `npm run create-release`. Follow the prompts to do a build, create a new release tag, and deploy the release tag.
```